graphene/src/document.rs

```diff
@@ -284,7 +285,7 @@ impl Document {
 					}
 				}
 			}
-			LayerDataType::Text(_) => layer.cache_dirty = true,
+			_ => layer.cache_dirty = true,
```

Comment:
Does this just make the whole match statement's various `layer.cache_dirty = true` statements redundant, rather than just setting it after the match statement? I'm not following the logic super closely but perhaps you can take a look if what I'm saying is accurate or not.